<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@3.3">
  <node class="cps.n.StepEndNode" plugin="workflow-cps@2.74">
    <parentIds>
      <string>66</string>
    </parentIds>
    <id>68</id>
    <startId>36</startId>
  </node>
  <actions>
    <wf.a.TimingAction plugin="workflow-api@2.37">
      <startTime>1579770328656</startTime>
    </wf.a.TimingAction>
    <wf.a.ErrorAction plugin="workflow-api@2.37">
      <error class="org.jenkinsci.plugins.workflow.steps.FlowInterruptedException" plugin="workflow-step-api@2.20">
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.cps.CpsBodyExecution.cancel(CpsBodyExecution.java:253)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.BodyExecution.cancel(BodyExecution.java:76)</trace>
          <trace>org.jenkinsci.plugins.workflow.cps.steps.ParallelStepExecution.stop(ParallelStepExecution.java:67)</trace>
          <trace>org.jenkinsci.plugins.workflow.cps.CpsThread.stop(CpsThread.java:308)</trace>
          <trace>org.jenkinsci.plugins.workflow.cps.CpsFlowExecution$6.onSuccess(CpsFlowExecution.java:1151)</trace>
          <trace>org.jenkinsci.plugins.workflow.cps.CpsFlowExecution$6.onSuccess(CpsFlowExecution.java:1140)</trace>
          <trace>org.jenkinsci.plugins.workflow.cps.CpsFlowExecution$4$1.run(CpsFlowExecution.java:907)</trace>
          <trace>org.jenkinsci.plugins.workflow.cps.CpsVmExecutorService$1.run(CpsVmExecutorService.java:37)</trace>
          <trace>java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)</trace>
          <trace>java.util.concurrent.FutureTask.run(FutureTask.java:266)</trace>
          <trace>hudson.remoting.SingleLaneExecutorService$1.run(SingleLaneExecutorService.java:131)</trace>
          <trace>jenkins.util.ContextResettingExecutorService$1.run(ContextResettingExecutorService.java:28)</trace>
          <trace>jenkins.security.ImpersonatingExecutorService$1.run(ImpersonatingExecutorService.java:59)</trace>
          <trace>java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)</trace>
          <trace>java.util.concurrent.FutureTask.run(FutureTask.java:266)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)</trace>
          <trace>java.lang.Thread.run(Thread.java:748)</trace>
        </stackTrace>
        <suppressedExceptions class="java.util.Collections$UnmodifiableRandomAccessList" resolves-to="java.util.Collections$UnmodifiableList">
          <c class="list"/>
          <list reference="../c"/>
        </suppressedExceptions>
        <result>
          <name>ABORTED</name>
          <ordinal>4</ordinal>
          <color>ABORTED</color>
          <completeBuild>false</completeBuild>
        </result>
        <causes class="java.util.Arrays$ArrayList">
          <a class="jenkins.model.CauseOfInterruption-array">
            <org.jenkinsci.plugins.workflow.steps.ExceptionCause>
              <t class="org.jenkinsci.plugins.workflow.steps.FlowInterruptedException">
                <stackTrace>
                  <trace>org.jenkinsci.plugins.workflow.cps.CpsFlowExecution.interrupt(CpsFlowExecution.java:1137)</trace>
                  <trace>org.jenkinsci.plugins.workflow.job.WorkflowRun$2.lambda$interrupt$0(WorkflowRun.java:367)</trace>
                  <trace>jenkins.security.ImpersonatingScheduledExecutorService$1.run(ImpersonatingScheduledExecutorService.java:58)</trace>
                  <trace>java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)</trace>
                  <trace>java.util.concurrent.FutureTask.run(FutureTask.java:266)</trace>
                  <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)</trace>
                  <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)</trace>
                  <trace>java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)</trace>
                  <trace>java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)</trace>
                  <trace>java.lang.Thread.run(Thread.java:748)</trace>
                </stackTrace>
                <suppressedExceptions class="java.util.Collections$UnmodifiableRandomAccessList" reference="../../../../../suppressedExceptions"/>
                <result>
                  <name>ABORTED</name>
                  <ordinal>4</ordinal>
                  <color>ABORTED</color>
                  <completeBuild>false</completeBuild>
                </result>
                <causes class="java.util.Arrays$ArrayList">
                  <a class="jenkins.model.CauseOfInterruption-array">
                    <jenkins.model.CauseOfInterruption_-UserInterruption>
                      <user>C5298959</user>
                    </jenkins.model.CauseOfInterruption_-UserInterruption>
                  </a>
                </causes>
              </t>
            </org.jenkinsci.plugins.workflow.steps.ExceptionCause>
          </a>
        </causes>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>